src/core_atmosphere/Makefile

```diff
@@ -47,17 +53,21 @@ physcore: mpas_atm_dimensions.o
 	( cd ../..; ln -sf ./src/core_atmosphere/physics/physics_wrf/files/*DATA* .)
 	( cd ../..; ln -sf ./src/core_atmosphere/physics/physics_noahmp/parameters/*TBL .)
 
-dycore: mpas_atm_dimensions.o $(PHYSCORE)
-	( cd dynamics; $(MAKE) all PHYSICS="$(PHYSICS)" )
+chemcore: mpas_atm_dimensions.o
+	( cd chemistry; $(MAKE) all )
+	( mkdir libchem; cd libchem; ar -x ../chemistry/libchem.a )
+
+dycore: mpas_atm_dimensions.o $(PHYSCORE) $(CHEMCORE)
+	( cd dynamics; $(MAKE) all PHYSICS="$(PHYSICS)"; $(MAKE) all CHEMISTRY="$(CHEMISTRY)" )
 
-diagcore: $(PHYSCORE) dycore
-	( cd diagnostics; $(MAKE) all PHYSICS="$(PHYSICS)" )
+diagcore: $(PHYSCORE) $(CHEMCORE) dycore
+	( cd diagnostics; $(MAKE) all PHYSICS="$(PHYSICS)"; $(MAKE) all CHEMISTRY="$(CHEMISTRY)" )
 
-utilities: $(PHYSCORE)
-	( cd utils; $(MAKE) all PHYSICS="$(PHYSICS)" )
+utilities: $(PHYSCORE) $(CHEMCORE)
+	( cd utils; $(MAKE) all PHYSICS="$(PHYSICS)"; $(MAKE) all CHEMISTRY="$(CHEMISTRY)" )
```

Comment:
Admittedly I haven't actually tried compiling yet, but do we really want to build the `all` target in the `utils` directory twice? Similarly for the diagnostics.

Reply:
Removed redundant MAKE all in dynamics, diagnostics, utils.